collections/_article/dev-snapshot-godot-4-4-dev-4.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "Dev snapshot: Godot 4.4 dev 4"
+excerpt: "After a short delay due to GodotCon, we are back with another exciting update."
+categories: ["pre-release"]
+author: Thaddeus Crews
+image: /storage/blog/covers/dev-snapshot-godot-4-4-dev-4.webp
+image_caption_title: ""
+image_caption_description: ""
+date: 2024-11-07 12:00:00
+---
+
+We are back roughly a month after the release of dev3 to bring you a host of new improvements. Many of our developers
+traveled to Germany in October for GodotCon and the yearly Godot contributor meeting. There they spent two days
+discussing some of the biggest technical challenges and puzzling out solutions.
+
+Many of the changes in this release are bug fixes that will be backported to Godot 4.3 and released in 4.3.1! So please
+test this release well so we can be confident with the changes and release 4.3.1 with them as soon as possible.
+
+Keep in mind that while we try to make sure each dev snapshot is stable enough for general testing, this is by
+definition a pre-release piece of software. Be sure to make frequent backups, or use a version control system such as
+Git, to preserve your projects in case of corruption or data loss.
+
+[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also try the [**Web editor**](https://editor.godotengine.org/releases/4.4.dev4/), [**XR editor**](https://www.meta.com/experiences/godot-game-engine/7713660705416473/), or the **Android editor** for this release (join the [Android editor testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds).
+
+-----
+
+*The cover illustration is from* [****](), *a ! It is developed in Godot 4.3 by [](). You can wishlist the game [on Steam]() and follow the developers on [Twitter]().*
+
+## Highlights
+
+In case you missed them, see the [4.4 dev 1](/article/dev-snapshot-godot-4-4-dev-1/), [dev2](/article/dev-snapshot-godot-4-4-dev-2/),
+and [dev 3](/article/dev-snapshot-godot-4-4-dev-3/) release notes for an overview of some key features which were
+already in that snapshot, and are therefore still available for testing in dev 3.
+
+Here are a few highlights that you might find particularly exciting!
+
+### Embed Game Window
```

Comment:
I'd make the title more clear:

```suggestion
### First steps towards an embedded game window
```

Reply:
I agree with changing this. I went with "Interactive in-game editing" to keep it simple, the explanation below explains that it is a first step to the fully embedded game window